live/global.ini

```diff
@@ -58255,9 +58255,9 @@ items_commodities_thorium_desc=Ein natürlich vorkommendes, schwach radioaktives
 items_commodities_thrust=Thrust
 items_commodities_thrust_desc=Dieser Cocktail aus Chemikalien verleiht dem Benutzer einen massiven Energieschub, erhöhte Geselligkeit, Euphorie und gesteigerte taktile Empfindlichkeit. Gerüchten zufolge wurde es als Basis für Energy-Drinks entwickelt und hat sich bei denen, die nach zusätzlicher Energie oder einem längeren "Feiern" suchen, als beliebt erwiesen. Aufgrund des Risikos von Herzinfarkt, tödlicher Dehydrierung und Krampfanfällen wurde die Mischung jedoch in den meisten Gebieten verboten. Benutzer sind an ihrem stockenden, wirren Sprechen, geröteter Haut und Schwitzen zu erkennen.
 items_commodities_tin,P=Tin
-items_commodities_tin_desc,P=Tin is a pliable, soft metal, with a silvery white-blue tint. It is conductive and resistance to corrosion.
+items_commodities_tin_desc,P=Zinn ist ein biegsames, weiches Metall mit einer silbrig-weiß-blauen Färbung. Es ist leitfähig und korrosionsbeständig.
```

Comment:
Zinn bitte in "Tin" zurückführen. Mach am besten Tin-Ore draus. Das sind Materialien/Gesteine die ich zu Items zähle. Darauf müssen wir generell auch achten. Ist noch nicht festgelegt.